omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -0,0 +1,89 @@
+package org.openmrs.module.idgen.rest.resource;
```

Comment:
You will also need to add the openmrs license header file.

Reply:
fixed